test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
Just remove everything under `test/` folder and replace minitest with rspec in your gemfile.